app.py

```diff
@@ -13,4 +13,19 @@ def print_board(board):
 
 def get_move():
     position = input("Enter the position for your next move in '#row# #column#' format\n")
-    return list(map(int, list(position.split())))
\ No newline at end of file
+    return list(map(int, list(position.split())))
+
+def alternate_turn():
+    global mark
+    if mark == "O":
```

Comment:
When you're working with a binary idea you should try to follow binary patterns. Here again we're flipping the bit to represent the icon state, which is separate from the icon itself.

Might make sense to do the binary operation against a 1/0 and have a map for the icon:

```
{0: "X", 1:"O"}
```